test/ios/OrtExtensionsUsage/Podfile

```diff
@@ -1,11 +1,13 @@
-platform :ios, '11.0'
+platform :ios, '12.0'
 
 target 'OrtExtensionsUsage' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
 
   # Pods for OrtExtensionsUsage
-  pod 'onnxruntime-c'
+  #pod 'onnxruntime-objc'  # can use this after 1.16 is released
```

Comment:
will need to use a pre-release version prior to 1.16, after the corresponding change in ORT to add the custom op registration API gets in.